src/algebra/field.lean

```diff
@@ -16,104 +16,48 @@ instance division_ring.to_domain [s : division_ring α] : domain α :=
       division_ring.mul_ne_zero (mt or.inl hn) (mt or.inr hn) h
   ..s }
 
-@[simp] theorem inv_one [division_ring α] : (1⁻¹ : α) = 1 := by rw [inv_eq_one_div, one_div_one]
-
-attribute [simp] inv_inv'
-
-lemma inv_inv'' [division_ring α] (x : α) : x⁻¹⁻¹ = x :=
-inv_inv'
-
-lemma inv_involutive' [division_ring α] : function.involutive (has_inv.inv : α → α) :=
-@inv_inv' _ _
-
-namespace units
-variables [division_ring α] {a b : α}
+/--Every field is a comm_group_with_zero.-/
+@[priority 10] -- see Note [lower instance priority]
+instance division_ring.to_group_with_zero {K : Type*} [division_ring K] :
+  group_with_zero K :=
+{ zero_mul := _,
+  mul_zero := _,
+  .. ‹division_ring K› }
 
-/-- Embed an element of a division ring into the unit group.
-  By combining this function with the operations on units,
-  or the `/ₚ` operation, it is possible to write a division
-  as a partial function with three arguments. -/
-def mk0 (a : α) (ha : a ≠ 0) : units α :=
-⟨a, a⁻¹, mul_inv_cancel ha, inv_mul_cancel ha⟩
-
-@[simp] theorem inv_eq_inv (u : units α) : (↑u⁻¹ : α) = u⁻¹ :=
-(mul_left_inj u).1 $ by rw [units.mul_inv, mul_inv_cancel]; apply units.ne_zero
-
-@[simp] theorem mk0_val (ha : a ≠ 0) : (mk0 a ha : α) = a := rfl
-
-@[simp] lemma mk0_coe (u : units α) (h : (u : α) ≠ 0) : mk0 (u : α) h = u :=
-units.ext rfl
+/--Every field is a comm_group_with_zero.-/
```

Comment:
I'm not sure how standard or otherwise this is, and I absolutely don't want to be prescriptive, but I think we're usually putting whitespace after the `/--` and before the `-/` in docstrings. :-)

Reply:
(I just checked: it's a 99-to-1 split. :-)

Reply:
While we're here, I'd add backticks around `comm_group_with_zero`. 😆

Reply:
Done. Thanks.